src/styles/cylc/_markdown.scss

```diff
@@ -1,33 +1,180 @@
-/**
- * Copyright (C) NIWA & British Crown (Met Office) & Contributors.
- *
- * This program is free software: you can redistribute it and/or modify
- * it under the terms of the GNU General Public License as published by
- * the Free Software Foundation, either version 3 of the License, or
- * (at your option) any later version.
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU General Public License for more details.
- *
- * You should have received a copy of the GNU General Public License
- * along with this program.  If not, see <http://www.gnu.org/licenses/>.
- */
-
-.markdown {
```

Comment:
This `.markdown` enclosure needs to be kept as otherwise the styles are applying in the wrong places, e.g. the command menu has got some spacing issues now:

![image](https://github.com/user-attachments/assets/5f23f12f-e282-4c16-8f49-6eff5fc0ece7)

Reply:
When I say "enclosure" I mean the `.markdown { ... }` wrapper around the rules

Reply:
(So that the new CSS only targets markdown, not other things in the UI)